packages/web-components/fast-components/src/radio-group/radio-group.template.ts

```diff
@@ -0,0 +1,14 @@
+import { html, slotted } from "@microsoft/fast-element";
+import { RadioGroup } from "./radio-group";
+
+export const RadioGroupTemplate = html<RadioGroup>`
+    <template
+        role="radiogroup"
+        ?aria-disabled="${x => x.disabled}"
```

Comment:
these two aren't true boolean attributes - they need to either not exist or be set to `"true"` or `"false"`